compat/poll/poll.c

```diff
@@ -462,7 +462,7 @@ poll (struct pollfd *pfd, nfds_t nfd, int timeout)
   if (timeout != INFTIM)
     {
       orig_timeout = timeout;
-      start = GetTickCount();
+      start = GetTickCount64();
```

Comment:
@linquize as @PhilipOakley pointed out, we dropped support for XP already after 2.10.0.

So arguing here that `GetTickCount64()` cannot be used before that support is dropped is kind of a bit funny.

Having said that, I am sympathetic to the needs of users who are stuck with XP for whatever reason. This sympathy does not allow me to magically make some time to get the support for XP back. But it allows me to spend a little more effort, *if* (and that is a very important *if*) there are developers who are actively working on getting XP support back and keep it working.

Are you this developer?